lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -899,6 +902,17 @@ SubExprInfo ExprEmitter::emitBinary(Operation *op, VerilogPrecedence prec,
   return {prec, signedness};
 }
 
+SubExprInfo ExprEmitter::emitVariadic(Operation *op, VerilogPrecedence prec,
+                                      const char *syntax, bool hasStrictSign) {
+  int i = 0;
+  for (auto v1 : op->getOperands()) {
```

Comment:
I'd recommend using the llvm::interleave template here from llvm/ADT/STLExtras.h